README.md

```diff
@@ -108,13 +77,7 @@ After you're finished please remove all the comments and instructions!
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
-
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Live Demo Link](replace-with-your-deployment-URL)
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/224508384-ab68482a-88e6-45ba-91e1-e5cf55377bc2.png)
![image](https://user-images.githubusercontent.com/93039359/224508377-327cdaf5-5c5c-4c7c-8c0a-30ea0e15f77d.png)


- [ ] It would be nice if you add Features in both sections shown above ⬆️ as suggested by the previous reviewer 😊 to make your readme look clean and professional 😊